cputemp/app.go

```diff
@@ -20,29 +21,29 @@ func getTemperature() float64 {
 	temp := out.String()
 
 	//temp=35.8'C
-	tempVal := temp[strings.Index(temp, "=") + 1 : len(temp) - 3 ]
+	tempVal := temp[strings.Index(temp, "=")+1 : len(temp)-3]
```

Comment:
VS code re-formats on save.  I'm not yet entirely convinced of its consistency in doing so.